apps/src/code-studio/levels/postOnLoad.js

```diff
@@ -49,6 +51,13 @@ export function postMilestoneForPageLoad() {
  * In practice, these will be identical.
  */
 export function onContinue() {
+  // Record the progress in redux - this will ensure progress is mirrored in the
+  // session storage if the user is logged out.
```

Comment:
Would it be helpful to indicate that this only applies to logged out users and to put a pointer to where progress is stored when the user is logged in?